gcip/addons/container/jobs/kaniko.py

```diff
@@ -54,34 +53,16 @@ def execute(
         dockerfile (str, optional): Name of the dockerfile to use. File is relative to context. Defaults to "Dockerfile".
         enable_push (bool, optional): Enable push to container registry, disabled to allow subsequent jobs to act on container tarball.
             Defaults to False.
+        docker_client_config (Optional[DockerClientConfig], optional): Creates the Docker configuration file base on objects settings,
+            used by crane to authenticate against given registries. Defaults to None.
```

Comment:
```suggestion
            used by crane to authenticate against given registries. Defaults to a `DockerClientConfig` with login to
            the official Docker Hub and expecting credentials given as environment variables `REGISTRY_USER` and
            `REGISTRY_LOGIN`.
```

Reply:
You markes this as resolved. Have you updated the documentation in unpushed commits?

Reply:
Yes.